shell.nix

```diff
@@ -0,0 +1,47 @@
+{ pkgs ? import <nixpkgs> {} }:
```

Comment:
Could you just add some documentation in a comment what this is, why one would care about it - most people aren't on a platform where this matters, so having clear documentation is important.